packages/react-components/react-combobox/src/components/Combobox/useCombobox.ts

```diff
@@ -52,10 +52,10 @@ export const useCombobox_unstable = (props: ComboboxProps, ref: React.Ref<HTMLBu
   const ignoreTriggerBlur = React.useRef(false);
 
   // popper
-  const popperOptions = {
+  const positioningOptions = {
     position: 'below' as const,
     align: 'start' as const,
-    offset: [0, 2] as Offset,
+    offset: 2,
```

Comment:
Should this change be to use the crossAxis, mainAxis offset instead of the single number?